src/anyon/qpu_interface.jl

```diff
@@ -49,17 +54,25 @@ const possible_status_list=[
     cancelled_status,
 ]
 
+function BasicAuthorization(
```

Comment:
Rename to snake_case and add a verb: `encode_to_basic_authorization` or similar.